.github/labeler.yml

```diff
@@ -19,6 +19,19 @@ CI/storybook-url:
 - "**/*.css"
 - "**/*.scss"
 
+feature/web3/wallet:
+- android/java/org/chromium/chrome/browser/app/domain/**/*
+- android/java/org/chromium/chrome/browser/crypto_wallet/**/*
+- components/brave_wallet/**/*
+- components/brave_wallet_ui/**/*
```

Comment:
`feature/web3/wallet` should be a superset of `feature/web3/wallet/core`.

It’s quite normal for us to add `feature/web3/wallet` to all wallet issues/PRs and then additionally use specific labels like `feature/web3/wallet/core` whenever applicable.

Reply:
Core is for native (c++) changes, `web3/wallet` is for all issues related to wallet (irrespective of platform/tech)